### DISCUSSION TITLE
Re-organize our repo folders / structure

### DISCUSSION THREAD

**Discussion by WebReflection** [pyscript/pyscript#2161] [Proposals]
I think [pyscript.sw](https://github.com/pyscript/pyscript/tree/main/pyscript.sw#readme) is completely redundant since we published and documented `mini-coi` as an easy fix ... these days we also have *sabayon* that accepts another kind of service worker and it's been 10 months that folder is not touched, but also it's nowhere documented or useful.

At the same time, once that's gone we can remove redundant prefixes inside the project and just have `core` instead of `pyscript.core` which is a tiny non-sense to me to have a project name prefix as a folder anyway, and we publish as `@pyscript/core` too, reflecting the `pyscript/core` structure I am after ... yay, nay, what do you think? To me it's a no-brainer:

  * drop *pyscript.sw*
  * rename *pyscript.core* folder as just *core*

Thanks.

/cc @ntoll @mchilvers @fpliger

**Comment by ntoll** [comment]
My 2c:

* We should rename `pyscript.core` to just `pyscript` (i.e. the name makes sense - it's where you find "PyScript").
* We should remove a bunch of other stuff we no longer need: e.g. `.readthedocs.yml`, duplicate `LICENSE`, `pyproject.toml`, `TROUBLESHOOTING.md` (could be folded into README).

**Reply by WebReflection** [reply]
> We should rename pyscript.core to just pyscript

we literally publish our package as `@pscript/core` and more non-core related things will land in pyscript as opt-in ... if we have a `pyscript/pyscript` folder I also think that's not ideal ... so, :-1: on renaming `pyscript.core` inside `pyscript` repo as `pyscript` for me, sorry.

**Reply by ntoll** [reply]
Ahh... yes, of course, it's like that because of the name we publish it under. Doh. Ignore me. ;-) Totally agree that `core` is the best name. :+1:

**Comment by ntoll** [comment]
Fixed by #2192
